packages/turf-union/index.js

```diff
@@ -27,22 +30,15 @@ import { GeoJSONReader, GeoJSONWriter, UnionOp } from 'turf-jsts';
  * //addToMap
  * var addToMap = [poly1, poly2, union];
  */
-function union() {
-    var reader = new GeoJSONReader();
-    var result = reader.read(JSON.stringify(arguments[0].geometry));
+function union(polygon1, polygon2) {
+    var geom1 = getGeom(polygon1);
+    var geom2 = getGeom(polygon2);
+    var properties = polygon1.properties || {};
```

Comment:
🤔 Tricky one... We should allow the user to override these properties from the `options` param.